src/dataflow/utilities/node.ts

```diff
@@ -1,3 +1,27 @@
+export const NodeTypes = [
```

Comment:
Seems like this could be an array of strings rather than objects, or perhaps even a string enum for stronger typing.

Reply:
I was thinking that this would eventually require more information (like icon information or some other node-specific details) so used an object rather than an array of strings.

Reply:
ok, that makes sense